MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -21,6 +21,7 @@ open class NavigationRouteOptions: RouteOptions {
             return $0
         }, profileIdentifier: profileIdentifier)
         includesAlternativeRoutes = true
+        shapeFormat = .polyline6
```

Comment:
Does MapboxNavigationNative handle polyline 5 format gracefully? If not, this is the first property being set here that the SDK strictly requires. Perhaps we should assert on the use of polyline 6 format or just go ahead and set it at the point of use in addition to here.